proteus/BC.py

```diff
@@ -0,0 +1,340 @@
+"""
```

Comment:
These should probably be moved to mprans since they're specific to two-phase air/water modeling.